src/controllers/auth.controller.js

```diff
@@ -93,6 +93,54 @@ class AuthController {
 		ResponseService.setSuccess(200, 'you have successful logged in', token);
 		return ResponseService.send(res);
 	}
+
+	/**
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} resetting the password
+*/
+	static async resetPassword(req, res) {
+		const decodedToken = await TokenService.verifyToken(req.query.token);
+		await UserService.updateUserByAttribute({ email: decodedToken.email }, {
+			password: BcryptService.hashPassword(req.body.password),
+			confirmPassword: BcryptService.hashPassword(req.body.confirmPassword)
+		});
+		if (req.body.password !== req.body.confirmPassword) {
```

Comment:
You should put this if condition in the middleware.

Reply:
done